finagle-mysql/src/main/scala/com/twitter/finagle/mysql/Client.scala

```diff
@@ -22,7 +22,7 @@ object Client {
       }
 
     def prepare(sql: String): PreparedStatement = new PreparedStatement {
-      def apply(ps: Any*): Future[Result] = factory() flatMap { svc =>
+      def apply(ps: Parameter*): Future[Result] = factory() flatMap { svc =>
```

Comment:
right, the style has changed.  since you're changing the line, could you please fix it?  we're changing the style lazily, so we ask that folks fix style as they change code.